test/nodes/test_prompt_node.py

```diff
@@ -51,6 +51,14 @@ def test_prompt_templates():
     assert p.prompt_text == "Here is some fake template with variable $baz"
 
 
+def test_prompt_template_repr():
+    p = PromptTemplate("t", "Here is variable $baz")
+    desired_repr = "PromptTemplate(name=t, prompt_text=Here is variable $baz, prompt_params=['baz'])"
+    assert p.__repr__() == desired_repr
+    assert p.__str__() == desired_repr
+    assert str(p) == desired_repr
```

Comment:
These two calls are fully equivalent, we can keep just one

```suggestion
    assert str(p) == desired_repr
```